roadie-kubernetes-cluster-access/test-values.yaml

```diff
@@ -0,0 +1,8 @@
+broker:
```

Comment:
It probably makse sense to put this into a directory with the three examples: aws, broker and other types.

Reply:
Oh that shouldn't have been there